Targets/Domain/Sources/Model/LoginModel.swift

```diff
@@ -45,6 +43,36 @@ public final class LoginModel: LoginModelProtocol {
 
 extension LoginModel {
     
+    // 최초 접속 1 회만, 호출됩니다.
+    public func requestAccessToken() async -> (String, Int)? {
+        let request = GetUserTokenRequest()
+        let result = await api.request(by: request)
+        
+        do {
+            switch result {
+                case .success(let data):
+                    let response = try JSONDecoder().decode(GetUserTokenResponse.self, from: data)
+                
+                LoginUserData.shared = .init(
+                    nickName: "",
+                    accessToken: response.accessToken,
+                    userId: response.userId,
+                    mail: ""
+                )
+                
+                return (response.accessToken, response.userId)
+                
+                case .failure(let error):
+                    print(error.description)
+                    return nil
+            }
+        } catch {
+            print(error.localizedDescription)
+        }
+        return nil
+    }
```

Comment:
```suggestion
    public func requestAccessToken() async -> Bool {
        let request = GetUserTokenRequest()
        let result = await api.request(by: request)
        
        do {
            switch result {
                case .success(let data):
                    let response = try JSONDecoder().decode(GetUserTokenResponse.self, from: data)
                
                LoginUserData.shared = .init(
                    nickName: "",
                    accessToken: response.accessToken,
                    userId: response.userId,
                    mail: ""
                )
                
                return true
                
                case .failure(let error):
                    print(error.description)
                    return false
            }
        } catch {
            print(error.localizedDescription)
            return false
        }
    }
```

data가 올바르게 내려오고 `LoginUserData.shared = .init(..)` 동작을 성공적으로 마친다면,  
관련 토큰과 userId 정보를 UI로 내려줄 필요가 없다고 생각됩니다!   
이후에 통신(Domain)은 모두 `LoginUserData.shared` 에 접근해서 토큰과 userId를 얻을 수 있습니다.   
위 수정을 제안해봅니다

Reply:
동의합니다!